gammapy/utils/modeling.py

```diff
@@ -55,17 +59,34 @@ class UnknownModelError(ValueError):
 
 
 class Parameter(object):
+    """
+    Class representing model parameters.
+
+    Parameters
+    ----------
+    name : str
+        Name of the parameter.
+    value : float or `~astropy.units.quantity`
```

Comment:
This is a typo. only ``float`` is accepted. For quantities there is a property with associated setter.